demisto_sdk/commands/common/hook_validations/id.py

```diff
@@ -397,3 +400,37 @@ def is_file_valid_in_set(self, file_path, file_type, ignored_errors=None):
                 is_valid = self._are_playbook_entities_versions_valid(playbook_data, file_path)
 
         return is_valid
+
+    def _is_pack_display_name_already_exist(self, pack_metadata_data):
+        """Check if the pack display name already exists
+        Args:
+            pack_metadata_data (dict): Dictionary that holds the extracted details from the given metadata file.
+        Returns:
+            bool. Whether the metadata file is valid or not.
+        """
+        new_pack_id = list(pack_metadata_data.keys())[0]
+        new_pack_name = pack_metadata_data[new_pack_id]['name']
+        for pack_id, pack_data in self.packs_set.items():
+            if new_pack_name == pack_data['name'] and new_pack_id != pack_id:
+                return False, Errors.pack_name_already_exists(new_pack_name)
```

Comment:
is the other way around (id exists but has a different name) legal? should we check for that?

Reply:
The pack_id is the name of the folder in our repo (there can be no duplicates)
pack_name - The display name of the pack.
The specific pack we do validation is also in id_set and therefore id_set already has its name, so we make sure that there is no other pack with the same name.

Reply:
then maybe we should change `pack_id` to `pack_folder_name` to clarify this.

Reply:
done.